channels/yandex-alice/src/main/kotlin/com/justai/jaicf/channel/yandexalice/api/AliceBotRequest.kt

```diff
@@ -59,12 +62,41 @@ data class Session(
     val messageId: Int,
     @SerialName("session_id")
     val sessionId: String,
+    @SerialName("skill_id")
+    val skillId: String,
+    val application: Application,
+    val user: User?
+) {
+    @Deprecated(
```

Comment:
Regarding the actual documentation this field is not deprecated

Reply:
https://yandex.ru/dev/dialogs/alice/doc/protocol-docpage/ Look at user_id field
```
Осторожно. Свойство перестает поддерживаться — вместо него следует использовать новое, полностью аналогичное свойство session.application.application_id.
```